src/main/java/net/minecraftforge/client/event/ScreenEvent.java

```diff
@@ -233,6 +240,32 @@ public PotionShiftEvent(Screen screen)
         }
     }
 
+    /**
+     * Fired to determine whether to render the potion indicators in the {@link EffectRenderingInventoryScreen inventory
+     * screen} in compact or classic mode.
+     *
+     * <p>This event is not {@linkplain Cancelable cancelable} and {@linkplain HasResult has a result}. </p>
```

Comment:
I think ending paragraph tags come up as warnings or errors when compiling the javadoc.

Reply:
I've double-checked, and ending paragraph tags do not emit warnings or errors according to `javadoc` (even with `-Xdoclint:all`).

Reply:
Ah, I see it. It's just bad html on another event that was generating the warnings.